blocks/rss-alexa-feature-block/features/rss/xml.test.js

```diff
@@ -0,0 +1,172 @@
+      itemTitle: 'headlines.basic',
+      pubDate: 'display_date',
+      itemCategory: '',
+      includePromo: true,
+      includeContent: 0,
+    },
+  })
+
+  expect(rss).toMatchSnapshot({
+    rss: {
+      channel: {
+        lastBuildDate: expect.stringMatching(
+          /\w+, \d+ \w+ \d{4} \d{2}:\d{2}:\d{2} \+0000/,
+        ),
+      },
+    },
+  })
+})
+
+it('returns Alexa template with custom values', () => {
+  const rss = Rss({
+    arcSite: 'demo',
+    globalContent: {
+      ...articles,
+    },
+    customFields: {
+      audioAvailable: "content_elements[?type=='audio'].streams[].url|[0]",
+      channelTitle: 'The Daily Prophet',
+      channelDescription: "All the news that's fit to print",
+      channelPath: '/arc/outboundfeeds/alexa/',
+      channelCopyright: '2020 The Washington Post LLC',
+      channelTTL: '60',
+      channelCategory: 'news',
+      channelLogo:
+        'https://arc-anglerfish-arc2-prod-demo.s3.amazonaws.com/public/JTWX7EUOLJE4FCHYGN2COQAERY.png',
+      itemTitle: 'headlines.seo || headlines.basic',
+      pubDate: 'display_date',
+      itemCategory: 'taxonomy.primary_section.name',
+      includePromo: true,
+      includeContent: 'all',
+    },
+  })
+
+  expect(rss).toMatchSnapshot({
+    rss: {
+      channel: {
+        lastBuildDate: expect.stringMatching(
+          /\w+, \d+ \w+ \d{4} \d{2}:\d{2}:\d{2} \+0000/,
+        ),
+      },
+    },
+  })
+})
+
+describe('check getURLType', () => {
+  it('return type is jpeg if url extension id .jpeg', () => {
+    const url = getURLType(
+      'https://cloudfront-us-east-1.images.arcpublishing.com/demo/GSVDM4QTMBBQZNY7GAVHKQGEFM.jpeg',
+    )
+    expect(url.type).toEqual('image/jpeg')
+  })
+
+  it('return type is JPG if url extension id .JPG', () => {
+    const url = getURLType(
+      'https://cloudfront-us-east-1.images.arcpublishing.com/demo/GSVDM4QTMBBQZNY7GAVHKQGEFM.JPG',
+    )
+    expect(url.type).toEqual('image/jpeg');
+  })
+  
+  it('return type is mp4 if url extension id .mp4', () => {
+    const vurl = getURLType(
+      'https://cloudfront-us-east-1.images.arcpublishing.com/demo/GSVDM4QTMBBQZNY7GAVHKQGEFM.mp4',
+    )
+    expect(vurl.type).toEqual('video/mp4')
+  })
+
+  it('show default return type  if url does not have extension', () => {
+    const vurl = getURLType(
+      'https://cloudfront-us-east-1.images.arcpublishing.com/demo/GSVDM4QTMBBQZNY7GAVHKQGEFM',
+    )
+    expect(vurl.type).toEqual('audio/mp3')
+  })
+
+  it('show return type is mp3 if url  have .mp3 AND parameter', () => {
+    const vurl = getURLType(
+      'https://clark.com/wp-content/uploads/2021/01/Ask-Clark_GameStop_2021.mp3?_=4',
+    )
+    expect(vurl.type).toEqual('audio/mp3');
+  })
+
+  it('show return label is Video if url  have .mp4 AND parameter', () => {
+    const vurl = getURLType(
+      'https://cloudfront-us-east-1.images.arcpublishing.com/demo/GSVDM4QTMBBQZNY7GAVHKQGEFM.mp4?_=4',
+    )
+    expect(vurl.label).toEqual('Video');
```

Comment:
Why are you testing label.  It's not used in the feed.  The test should be on type.